src/types/token.type.ts

```diff
@@ -0,0 +1,8 @@
+type Token = {
+    name: string;
+    decimals: number;
+    symbol: string;
+    contractAddress: string;
+};
+
+export default Token;
```

Comment:
Is this type used for storing the values retrieved from the contract?

Reply:
Yes is the type that is used to represent the token object, I think that is the one that needs to be used across the other modules but we could discuss it.

Reply:
I found [where we use it](https://github.com/rsksmart/rif-relay-server/pull/39/files#diff-4d7169a5d6399aede3f48a43d56bcbaf00efd5b168aa03d04e7298ceca6cd60aR4)